tonic/src/status.rs

```diff
@@ -470,9 +521,11 @@ impl fmt::Display for Status {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         write!(
             f,
-            "grpc-status: {:?}, grpc-message: {:?}",
+            "grpc-status: {:?}, grpc-message: {:?}, grpc-status-details-bin: {:?}, metadata: {:?}",
             self.code(),
-            self.message()
+            self.message(),
+            base64::encode_config(self.details(), base64::STANDARD_NO_PAD),
```

Comment:
I don't think we need to base64 here since from the user's pov they don't really care how it is encoded on the wire but only how they passed it in.

Reply:
My motivation for opting for base 64 here was this: in your previous message you used header names, not struct field names. E.g. `grpc-status: {:?}`, `grpc-message: {:?}`. This is what will be sent to the wire, not what user provided to the struct itself.

If this message has to do more with user perspective, then maybe we should use struct field names instead of header names everywhere? Like `status: {:?}, message: {:?}, details: {:?}, metadata: {:?}"`?

Reply:
Yeah, that seems like a totally fine thing to do! I think I like that!

Reply:
Done